documentation/writing_yaml_plans.md

```diff
@@ -277,6 +277,11 @@ the targets specified with the `targets` field. For more information about
 `apply_prep` see the [Applying manifest blocks](applying_manifest_blocks.md#)
 section.
 
+> 🔩 **Tip:** Because each `resources` step executes the `apply_prep` function
```

Comment:
Did we benchmark this? How many resources steps does it take to have a noticeable impact? I'd expect apply_prep to go pretty quick once the agent is installed and facts are (remotely) cached

Reply:
I'll just remove that bit.